doc/sonic-sub-port-intf-hld.md

```diff
@@ -105,13 +105,13 @@ For APPL_DB and STATE_DB, we do not introduce new tables for sub port interfaces
     }
 },
 ```
-A key in the DOT1Q_INTERFACE table is the name of a sub port, which consists of two sections delimited by a "." (symbol dot). The section before the dot is the name of the parent physical port or port channel. The section after the dot is the dot1q encapsulation vlan id.
+A key in the SUB_INTERFACE table is the name of a sub port, which consists of two sections delimited by a "." (symbol dot). The section before the dot is the name of the parent physical port or port channel. The section after the dot is the dot1q encapsulation vlan id.
 
 mtu of a sub port interface should be no greater than that of the parent physical port or port channel. In the case field "mtu" is absent in the config_db.json file, a sub port interface inherits the mtu from its parent physical port or port channel.
 
 Example configuration:
 ```
-"DOT1Q_INTERFACE": {
+"SUB_INTERFACE": {
```

Comment:
better to be dot1q_subinterface  as we could have other typwa of subinterface

Reply:
How about VLAN_SUB_INTERFACE? do we need DOT1Q for some reason?

Reply:
sounds good